### DISCUSSION TITLE
California & Omega theme not present in template list

### DISCUSSION THREAD

**Discussion by nazrulcsebd** [primefaces/community#1868] [PrimeNG] [answered]
Dear PrimeNG team, I bought the California & Omega theme and used it in my development. But recently the California and Omega theme is not present in the template list. Only some templates are there. If you are not given the updation of those two themes for the new release of Angular then what we are doing now? Please give us a guide on what we are doing now.

**Comment by WahidAbdullah** [comment]
Those who use California & Omega theme they really face a great Challenge for the new release of Angular. So please give us a guideline how can we cope up with Angular new releases and go forward in our development.

**Reply by syssarkar** [reply]
provide solution as soon as possible

**Comment by Rahad140214** [comment]
Excellent job....

**Comment by syednurulislam** [comment]
We also need the clear guideline for up gradation of angular in california theme. If the team do not update it, what will be our future policy? Please take it under your consideration.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Omega and California are retired templates, and they will no longer receive updates. For more info about retired templates [please see](https://github.com/orgs/primefaces/discussions/1802). However, we are preparing a guide for users to be able to update the templates themselves, eliminating the need to wait for an official update. This guide will be added to [primeng.org](http://primeng.org/).
